src/atoms/Text/Text.native.js

```diff
@@ -32,6 +32,12 @@ const styles = {
       return getLineHeight(theme, size);
     }
   },
+  textAlign({ align }) {
+    if (align == 'inherit' || align == 'initial') {
```

Comment:
Shouldn't we use ```===```?
```suggestion
    if (align === 'inherit' || align === 'initial') {
```

Reply:
Don't we have this check as eslint rule? 🤦‍♂

Reply:
We do. Something wrong with Shridhar's machine. I guess I need to now run linters in CI as well